app.py

```diff
@@ -162,19 +166,14 @@ def registerForm():
     password = request.form.get('password')
     first_name = request.form.get('first_name')
     email = request.form.get('email')
-    existing_user = user_data.query.filter_by(username=username).first()
-    existing_email = user_data.query.filter_by(email=email).first()
-
+    # existing_user = user_data.query.filter_by(username=username).first()
+    # existing_email = user_data.query.filter_by(email=email).first()
 
 #     if (existing_email and existing_user):
 #         return redirect('/login')
 
 
-    bcryptRounds = int(os.getenv('BCRYPT_ROUNDS'))
-    if bcryptRounds == 'None':
-        print("Defaulting bcryptRounds (error)")
-        #bcrypt rounds are too high
-        bcryptRounds = 20000 # If bcrypt rounds is not found, falls back to default value of 20k
+    bcryptRounds = int(os.getenv('BCRYPT_ROUNDS', 4)) # second parameter is the default fallback

```

Comment:
question: are we going to make it 4 just in general since free render is slow asf?